dali/operators/reader/loader/nemo_asr_loader_test.cc

```diff
@@ -97,6 +76,26 @@ TEST(NemoAsrLoaderTest, ParseManifest) {
   EXPECT_EQ("path/to/audio1.wav", entries[0].audio_filepath);
 }
 
+TEST(NemoAsrLoaderTest, ParseNonAsciiTransript) {
+  {
+    std::stringstream ss;
+    ss << R"code({"audio_filepath": "path/to/audio1.wav", "duration": 1.45, "text": "это проверка"})code" << std::endl;
+    std::vector<NemoAsrEntry> entries;
+    detail::ParseManifest(entries, ss);
+    ASSERT_EQ(1, entries.size());
+    EXPECT_EQ(std::string(u8"это проверка"), entries[0].text);
+  }
+
+  {
+    std::stringstream ss;
+    ss << R"code({"audio_filepath": "path/to/audio1.wav", "duration": 1.45, "text": "这是一个测试"})code" << std::endl;
```

Comment:
Perhaps we should get the utf-8 bytes and store it as a `char` array (providing the characters' numerical values) - otherwise it's prone to breaking when the file is transcoded.